CHANGELOG.md

```diff
@@ -70,6 +70,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   to fetch during clone. If present, the first matching branch is used as the
   working-copy parent.
 
+* `jj status` supports hiding the list of untracked files with flag `--hide-untracked`
+  and config option `ui.status-hide-untracked`.
```

Comment:
I can see the use for the flag e.g. for scripting. When would one want the config? Why not add the paths to the `.gitignore` when you get tired or seeing them?

Reply:
Hmm, a config for people with bad habits? :) I suppose if it's common that people have this bad habit, this might make sense.

Serious question: why do you keep those untracked files in your working copy instead of adding them to your .gitignore when you see them?

Reply:
They're mostly random temporary files with unpredictable names for which it makes little sense to add to anyone's gitignore. E.g. I run `./program --flag > log-a` fix something `./program --no-flag > log-b` `diff log-a log-b` and then forget about the whole thing. If I was starting a new project I might find it useful that jj encourages me to be better organized and delete these quicker. But to adopt jj in my existing projects, I really need some way to ignore the literally multiple screenfuls of untracked files.